### PR TITLE
tests/kola: add test for separate /var mount

### PR DESCRIPTION
**Pull request by jlebon**
We regressed on this a few times. Let's add a test for it.

### CODE REVIEW

File: tests/kola/var-mount/test.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+set -xeuo pipefail
+
+# restrict to qemu for now because the primary disk path is platform-dependent
```
Comment (cgwalters) [inline tests/kola/var-mount/test.sh:4]: Not strictly related to this PR but I think we really need to introduce something like `/dev/disk/coreos-root` or so.

**Review by cgwalters — APPROVED** [review]
LGTM

**Comment by jlebon** [comment]
CI here will fail until https://github.com/ostreedev/ostree/pull/2186 reaches FCOS.

**Comment by cgwalters** [comment]
I rebased :surfing_man: this now that we have https://github.com/coreos/fedora-coreos-config/pull/596

**Comment by cgwalters** [comment]
OK hmm that last CI run did successfully get the latest ostree and the new test passed; the upgrade run just flaked downloading an image.

**Comment by cgwalters** [comment]
Hooray, green here!  Let's make sure we don't break this again.

**Pull request merged**